rollup.config.js

```diff
@@ -871,10 +871,10 @@ export default function rollup(options) {
 async function triggerLiveReload(appDir) {
   // tickle live reload by touching the server entry
   let serverEntryPaths = [
+    "entry.server.ts",
```

Comment:
Can we add a similar comment to this line explaining that `React.createElement` can be used instead of JSX?